autoload/SpaceVim.vim

```diff
@@ -76,7 +76,7 @@ let g:spacevim_unite_leader            = '\f'
 ""
 " Denite work flow leader of SpaceVim. Default is `F`.
 " Set to empty to disable this feature, or you can set to another char.
-let g:spacevim_denite_leader            = 'F'
+let g:spacevim_denite_leader            = '\f'
```

Comment:
This does not appear to be related to 'vimcompatible' changes, but i assume that is intentional.

Don't forget to update the comment on line 77. Likewise, 73 and 75 appear to not match.